torchtext/data/batch.py

```diff
@@ -19,6 +20,7 @@ class Batch(object):
 
     def __init__(self, data=None, dataset=None, device=None):
         """Create a Batch from a list of examples."""
+        warnings.warn('Batch class will retire in 0.8.0 release', RuntimeWarning)
```

Comment:
According to [this comment](https://github.com/pytorch/audio/pull/523#discussion_r405673957), PyTorch's convention for deprecation is `UserWarning`.

Reply:
Thanks